pkg/sidecar-controller/framework_test.go

```diff
@@ -171,6 +172,30 @@ func (r *snapshotReactor) React(action core.Action) (handled bool, ret runtime.O
 
 	// Test did not request to inject an error, continue simulating API server.
 	switch {
+
+	case action.Matches("patch", "volumesnapshotcontents"):
+		patchAction := action.(core.PatchActionImpl)
+		patch := patchAction.GetPatch()
+		var content *crdv1.VolumeSnapshotContent
+		var newcontent *crdv1.VolumeSnapshotContent
+		err := json.Unmarshal(patch, newcontent)
+
+		_, err = createContentPatch(content, newcontent)
```

Comment:
neither `content` nor `newcontent` should be nil.  Otherwise there's nothing that needs to be merged.

Reply:
@xing-yang sure, i will updates the switch case functionality accordingly